cg/src/domains/subjects/data-shares.service.js

```diff
@@ -8,8 +8,10 @@ class DataShareService {
     return db('data_shares')
       .where({ subject_id: subjectId })
       .map(dataShare => ({
+        id: dataShare.id,
         name: dataShare.name,
-        url: `${process.env.url}/api/subjects/data-shares/share?token=${dataShare.token}`
+        token: dataShare.token,
+        url: `${process.env.ROOT_URL}/api/subjects/data-shares/share?token=${dataShare.token}`
```

Comment:
I think we should use the `CONTROLLER_URL` here instead or we need to update the quick start scripts.

Reply:
The config generated by `node setup.js` has the value `CONTROLLER_URL=http://cg:8082`. Is generating in different way on production deploys?

Reply:
No, there's no automation yet for production deployments, but our demo API has manually generated proper env vars. If you change it to `ROOT_URL` we'll need to include it in `setup.js` and ask DevOps for update on Kubernetes.

Reply:
Let's do it `CONTROLLER_URL` then! 👍